accname/name/comp_name_from_content.html

```diff
@@ -2,35 +2,183 @@
 <html>
 <head>
   <title>Name Comp: Name From Content</title>
+  <meta charset="utf-8">
   <script src="/resources/testharness.js"></script>
   <script src="/resources/testharnessreport.js"></script>
   <script src="/resources/testdriver.js"></script>
   <script src="/resources/testdriver-vendor.js"></script>
   <script src="/resources/testdriver-actions.js"></script>
   <script src="/wai-aria/scripts/aria-utils.js"></script>
+  <style type="text/css">
+
+    .simple-before::before {
+      content: " before "; /* [sic] leading and trailing space on most content props, unless marked. */
```

Comment:
@cookiecrook Are the leading/trailing spaces used to test correct prepending/appending of content?